Source/CarthageKit/Cartfile.swift

```diff
@@ -286,10 +286,10 @@ extension Dependency: Scannable {
 		} else if scanner.scanString("binary", into: nil) {
 			parser = { urlString in
 				if let url = URL(string: urlString) {
-					if url.scheme == "https" {
+					if url.scheme == "https" || url.scheme == "file" {
 						return .success(self.binary(url))
 					} else {
-						return .failure(ScannableError(message: "non-https URL found for dependency type `binary`", currentLine: scanner.currentLine))
+						return .failure(ScannableError(message: "non-https or non-file URL found for dependency type `binary`", currentLine: scanner.currentLine))
```

Comment:
Can you change this to say `non-https, non-file URL`?

`non-https or non-file URL` makes it sound like the the URL needs to be both https and a file URL.